main.go

```diff
@@ -107,6 +107,7 @@ var (
 		"vetshadow":   "go tool vet --shadow ./*.go:PATH:LINE:MESSAGE",
 		"gofmt":       `gofmt -s -d -e .:^diff\s(?P<path>\S+)\s.+\s.+\s.+\s@@\s-(?P<line>\d+)`,
 		"gotype":      "gotype -e {tests=-a} .:PATH:LINE:COL:MESSAGE",
+		"goimports":   `goimports -d .:^diff\s(?P<path>\S+)\s.+\s.+\s.+\s@@\s-(?P<line>\d+)[\S\s]+import`,
```

Comment:
I think this would be better as `goimports -l ./*.go:^(?P<path>[^\n]+)$`. The `./*.go` is necessary to prevent recursion and using just `-l` is faster as it doesn't have to compute the diff.